windows/src/desktop/kmshell/xml/menu-frame.js

```diff
@@ -28,10 +28,17 @@
     save_state();
 
     if(!loading_state) {
-      var q = document.getElementById('content_'+c.menu_name).getElementsByTagName('span');
+      let q = null;
+      if (c.menu_name.includes("keyboardlist")) {
+        q = document.getElementById('content_'+c.menu_name).getElementsByClassName('list_item expanded');
+      }
+      // if none are expanded select the first item
```

Comment:
The comment is a bit misleading - we're potentially getting more than one item. Maybe this comment should be moved to just before the `for` loop and changed to `// select the first list item` ?